lib/index.ts

```diff
@@ -29,13 +43,24 @@ const noop = () => {};
  *  * /Dockerfile -> Dockerfile
  *  * Dockerfile -> Dockerfile
  *  * ./a/b/Dockerfile -> a/b/Dockerfile
+ *  * foo/bar/ -> foo/bar/
+ *  * foo/bar/ -> foo/bar
+ * See additional input/output examples in tests.ts
  */
 export function normalizeTarEntry(name: string): string {
-	const normalized = path.normalize(name);
-	if (path.isAbsolute(normalized)) {
-		return normalized.substr(normalized.indexOf(path.sep) + 1);
+	if (!name) {
+		return '';
+	}
+	let normalized = normalize(name);
+	// remove any leading or trailing slashes
+	if (normalized.startsWith('/')) {
```

Comment:
I don't know how much other code in here would benefit from `lodash`, but certainly `return _.trim(normalized, '/')` would save the next 7 lines.

Reply:
Nice! There was also a global `noop` function that I have now replaced with `_.noop`. (In the past I thought that `noop` alone wasn't enough reason to add lodash as a dependency, but `_.trim` tipped the balance for me.) :-)